### DISCUSSION TITLE
enabling crontab

### DISCUSSION THREAD

**Discussion by DNCD** [simple-login/app#1639] [Self-hosting question]
How can I enable schedule crontab instead of a container with args.job running every minute? I was looking but couldn't find any documents regarding crontab. I'm trying to run HIBP, but it runs every minute when I use `python /code/cron.py -j check_hibp` inside a container. Thanks

**Comment by nguyenkims** [comment]
@DNCD the cronjob isn't running every minute but it adds a delay before each api call to HIBP to respect the rate limit.

**Reply by DNCD** [reply]
The container auto restart every minute.
